makefiles/vars.inc.mk

```diff
@@ -72,6 +72,7 @@ export WERROR                # Treat all compiler warnings as errors if set to 1
 export GITCACHE              # path to git-cache executable
 export GIT_CACHE_DIR         # path to git-cache cache directory
 # FLASHER                    # The command to call on "make flash".
+# PROG_PORT                  # The port to connect the FLASHER and DEBUGGER
```

Comment:
"The serial port" ?

Reply:
How about I changed it for `The device to connect`? At list in linux this always maps to some kind of device under `/dev`. I used `PROG_PORT` to keep the link with the fact that `PORT`, but in the later the name makes more sense since it is always a `serial device port`. For the progamer it can be a serial device port or a usb device or other. For these the term `PORT` is actually not always accurate. Maybe `DEV_PROG` or `PROG_DEV` might be a better name?

Reply:
`PROG_DEV`

Reply:
@aabadie do you agree?

Reply:
+1 for `PROG_DEV`

Reply:
Done.